discojs/discojs-core/src/dataset/data/preprocessing/text_preprocessing.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { TEXT_PREPROCESSING } from './index.js'
+import { expect } from 'chai'    
+
+import type { Task } from '../../../index.js'
+import * as tf from '@tensorflow/tfjs'
+
+describe('text preprocessing', function () {
+  const [tokenize, leftPadding] = TEXT_PREPROCESSING
+  // Use a function to create different task object for each test (otherwise the tokenizer gets cached)
+  const initMockTask: () => Task = () => {
```

Comment:
I find the function notation to be much more readable.

```suggestion
  function initMockTask(): Task {
```